src/Controller/HomeController.php

```diff
@@ -22,4 +23,34 @@ public function home(NewsRepository $newsRepository)
     {
         return ['posts' => $newsRepository->findLatest(self::ITEMS_ON_HOMEPAGE)];
     }
+
+    /**
+     * @Route("/about_mixDesk", name="mixDesk_guide")
+     * @Template("mixDesk.html.twig")
+     *
+     */
+    public function mixDeskGuide()
+    {
+        return [];
+    }
+
+    /**
+     * @Route("/about_genres", name="genres_guide")
+     * @Template("genres_guide.html.twig")
+     *
+     */
+    public function genresGuide()
+    {
+        return [];
+    }
+
+    /**
+     * @Route("/contacts", name="contacts")
+     * @Template("contacts.html.twig")
+     *
```

Comment:
@daria-sieroshtan 
this line is unnecessary, and there are 3 of them in a row :)